puppet/modules/freight/manifests/user.pp

```diff
@@ -73,7 +73,7 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0644',
-    content => epp("${module_name}/${vhost}-HEADER.html.epp", { 'stable' => $profiles::web::stable }),
+    content => epp("${module_name}/${vhost}-HEADER.html.epp", { 'stable' => $web::vhost::deb::stable }),
```

Comment:
It feels a bit better to make it an explicit parameter rather than refer to some random other class variable.